client/extensions/woocommerce/woocommerce-services/views/live-rates-carriers-list/carriers-list.js

```diff
@@ -32,14 +32,9 @@ const Actions = localize( ( { translate } ) => {
 const CarrierDiscount = localize( ( {
```

Comment:
I wondered whether having a whole component/function for this is worth it at this point - but whatevs 🤷